operations/host_spawn.go

```diff
@@ -1018,6 +1028,29 @@ func printHosts(hosts []*restModel.APIHost) {
 	}
 }
 
+func printHostsJSON(hosts []*restModel.APIHost) {
+	var output strings.Builder
+	output.WriteString("[")
+	for i, h := range hosts {
```

Comment:
Can we try to simply marshal the host list to JSON and print it like the way we do here? https://github.com/evergreen-ci/evergreen/blob/main/operations/patch_list.go#L88 This seems cleaner than hardcoding.